draft-ietf-masque-connect-udp-listen.md

```diff
@@ -164,6 +166,12 @@ and therefore is to be ignored). This document does not define any parameters
 for the connect-udp-listen header field value, but future documents might define
 parameters. Receivers MUST ignore unknown parameters.
 
+
+# The proxy-public-address Response Header Field {#response}
+Upon accepting the request, the "proxy-public-address" response header,
+MUST contain the IP and the UDP port which are allocated to the client by the
+proxy in the format {IP-Address}:{UDP-Port}
+
```

Comment:
Another thought is about address changes mid-stream. We could support those by using a capsule instead of a header, but that might be overkill. Maybe we should acknowledge the limitation though

Reply:
An address change would break connectivity, right? If I've told other devices my "public" IP through the proxy and that changes, I need to start over. That should be a different request.

Reply:
Not necessarily. If a new address is added, then the layer above UDP could conceivably communicate that, and then migrate traffic over to using it.

But this is probably too niche and not worth the complexity.

Reply:
Can we also add a sentence here to mention the fact that addresses cannot be updated mid-stream? Perhaps "Note that since the addresses are conveyed in HTTP response headers, a subsequent change of addresses on the proxy cannot be conveyed to the client."